lib/components/AlbumScreen/album_screen_content.dart

```diff
@@ -22,10 +19,12 @@ class AlbumScreenContent extends StatefulWidget {
     Key? key,
     required this.parent,
     required this.children,
+    required this.playableChildren,
   }) : super(key: key);
 
   final BaseItemDto parent;
   final List<BaseItemDto> children;
+  final List<BaseItemDto> playableChildren;
```

Comment:
So in #484 I introduces some new classes/types to attach more metadata onto list items (i.e. `FinampQueueItem`), maybe that's the way to go here as well? Having two separate lists of `BaseItemDto` seems like it would quickly become hard to manage.  
But if you think the API is fine, you can leave it as it is :)

Reply:
After thinking about this for awhile, I decided to leave this basically as-is.  This class immediately starts maintaining two lists anyway because child elements need a display list of just children in that disc as well as another list with the entire album.  So I just slightly improved the names and left it at that.